analyzer/Makefile

```diff
@@ -3,8 +3,8 @@ all: lint test
 include ../tools/python.mk
 
 PIP_INSTALL := --editable .[dev]
-PYLINT_ARG := libretime_analyzer tests
-MYPY_ARG := libretime_analyzer tests
+PYLINT_ARG := libretime_analyzer tests || true
```

Comment:
We initially didn't make the CI fail on linting because the number of linting error in the code required too much work and code change.

While shared and tools does not generated any linting errors, the others package still do. We need to slowly improve the code quality and then allow CI to fail a package at the time.

Reply:
And note that I actually simply moved the `|| true` failsafe from `tools/python.mk` to each app makefile so we can have a fine grained configuration on which app should make the CI fail and which shouldn't yet.

Reply:
Ah I see. I thought it was going the other way - from not ignoring failures to ignoring them.

Reply:
Yes, when pushing multiple feat/fix commits in a single PR, I try to build clean git history for you (any) to review. Checking one commit after the other helps understand the logic of the changes instead of having the PR commits squash into a single diff view.